### DISCUSSION TITLE
Suddenly upload api has stopped working for SauceRest

### DISCUSSION THREAD

**Discussion by akumar2207** [saucelabs/saucerest-java#350] [General]
![73199E29-753C-41E3-8F56-EB11B458CE98](https://user-images.githubusercontent.com/46046732/228824668-38109fe6-531d-4c0b-a134-e511c5b7de56.jpeg)
Hi Team ,
I have attached the screenshot of my code .
It was working since yesterday and suddenly started getting exception
.
Used maven dependency:
SauceRest: 2.0.4
![7117899F-A07E-44C5-B7E1-F803AC80C935](https://user-images.githubusercontent.com/46046732/228824370-148be907-0f4d-40cc-b38b-421ac4da22d4.jpeg)
![Uploading 8BBF0991-7CCC-46A3-85B3-D7DFED1AFB69.jpeg…]()

**Comment by sceiler** [comment]
Thank you. This is going to be handled in https://github.com/saucelabs/saucerest-java/issues/351 and therefore I will close this discussion as a duplicate.
